frontend/src/components/UserUpdateSfdsAuth.vue

```diff
@@ -189,6 +190,12 @@ export default {
     }
   },
   computed: {
+    filteredSfdsAuthorizations() {
+      return this.sfdsAuthorizations.filter(item => this.selectedClient.clientId=='HSCIS'?item.u.includes("hscis"):!item.u.includes("hscis"));
```

Comment:
I'm not familiar with SFDS permissions, but I suspect the data that produced this screenshot won't be replicated in production. If a user has "combination use" record that included hscis, you would only see that record with HSCIS selected. It seems to me that after this change it won't be possible to create a combination record that includes the HSCIS use, as the HSCIS use will be selectable if and only if the HSCIS client is selected. So this is good, right @tschia ?

![image](https://user-images.githubusercontent.com/1767127/146816612-d51ac03d-56c3-4604-83cc-d9f0470a6e39.png)